lib/wanda/execution/server.ex

```diff
@@ -44,13 +44,44 @@ defmodule Wanda.Execution.Server do
           targets: targets
         } = state
       ) do
-    :ok =
+    case Wanda.Messaging.ProcessCache.get(execution_id) do
```

Comment:
The thing here is this:
I was getting many times the same message, so I thought about at least once message delivery by rabbit.

The point is: what happens if we get an `ExecutionRequested` event (for the same execution id) more than once?

I believe that we need to be able to know whether an execution is already started, start it if needed, or just ignore if already running.

I thought of a simple cache where we track started executions.

Doubts are:
- maybe I am missing some other nice way with supervisors/genservers
- if using a cache would be acceptable, I am not sure doing this operation in the `Server` is fine, another option is doing it somewhere else, let's say closer to the consumer layer
- any other thing I overlooked :eyes: